packages/dart_firebase_admin/lib/src/app/credential.dart

```diff
@@ -1,5 +1,7 @@
 part of '../app.dart';
 
+Platform platform = const LocalPlatform();
```

Comment:
This shouldn't be public. At the very least, it needs to be `@internal` and the `lib/` export needs to `hide` it.

Reply:
Removed.